File: backend/Chess.Common/Helpers/ClientEvent.cs
namespace Chess.Common.Helpers
{
    public enum ClientEvent
    {
        [StringValue("invocationReceived")]
        Invocation,
        [StringValue("invocationAccepted")]
        InvocationAccepted,
        [StringValue("invocationDismissed")]
        InvocationDismissed,
        [StringValue("invocationCanceled")]
        InvocationCanceled,
        [StringValue("moveCommitted")]
        MoveCommitted,
        [StringValue("check")]
        Check,
        [StringValue("mate")]
        Mate,
        [StringValue("resign")]
        Resign,
        [StringValue("draw")]
        Draw
    }
}

File: frontend/src/app/core/models/signalR/clientEvent.ts
export enum ClientEvent {
    InvocationReceived = 'invocationReceived',
    InvocationAccepted = 'invocationAccepted',
    InvocationDismissed = 'invocationDismissed',
    InvocationCanceled = 'invocationCanceled',
    MoveCommitted = 'moveCommitted',
    Check = 'check',
    Mate = 'mate',
    Resign = 'resign',
    Draw = 'draw'
}